src/utils/Tests.java

```diff
@@ -4,4 +4,9 @@ public class Tests {
 
     //creating test
     //tc145
+
+
+    // working tc 124
+
+    //.......
```

Comment:
This part is doing the same thing as the cool method. Maybe use the util method.